WordPress/src/main/java/org/wordpress/android/ui/stories/prefs/StoriesPrefs.kt

```diff
@@ -17,7 +18,9 @@ class StoriesPrefs @Inject constructor(
     private val context: Context
 ) {
     companion object {
+        private val KEY_STORIES_SLIDE_INCREMENTAL_ID = "incremental_id"
         private val KEY_PREFIX_STORIES_SLIDE_ID = "story_slide_id-"
+        private val KEY_PREFIX_TEMP_MEDIA_ID = "t-"
         private val KEY_PREFIX_LOCAL_MEDIA_ID = "l-"
         private val KEY_PREFIX_REMOTE_MEDIA_ID = "r-"
```

Comment:
These values could be marked as constants :) using `const`

Reply:
👍 done in 695fed6d0b